py/visdom/server.py

```diff
@@ -601,8 +601,9 @@ def wrap_func(handler, args):
         p = handler.state[eid]['jsons'][args['win']]
 
         if not (p['type'] == 'text' or
-                p['content']['data'][0]['type'] == 'scatter'):
-            handler.write('win is not scatter or text')
+                p['content']['data'][0]['type'] in ['scatter', 'custom']):
```

Comment:
Note: This change is also in https://github.com/facebookresearch/visdom/pull/377#pullrequestreview-129403199

You may run into merge conflicts when landing these.